### PR TITLE
added Docker files

### PR DESCRIPTION
**Pull request by tarekziade**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+web:
+  build: .
+  links:
+   - redis
+  ports:
+   - "8000:8000"
+redis:
+  image: redis
```
Comment (leplatrem) [inline docker-compose.yml]: that makes Redis available on localhost:6379 ? No change needed in app conf ?

Comment (tarekziade) [inline docker-compose.yml]: Yes that's the default port

**Comment by ametaireau** [comment]
Ah okay!

Cool that we have that. We've changed to use postgres as a default to store everything. So if we want to have docker use postgres, we probably want to pull postgres in addition to redis and create the default table.

**Comment by ametaireau** [comment]
(I've rebased this PR on top of master)

**Pull request merged**